### ISSUE TITLE
Some Sampler choices doesnt show on Civitai.

### ISSUE THREAD

**Issue by Puddz** [alexopus/ComfyUI-Image-Saver#10]
As title says. Some of the samplers seem to have a different name compared to what they called on civitai, and civitai doesn't seem to pick it up. 
Like Euler Ancestral sampler shows up as Euler A on prompt readers and stuff, but civitai doesnt seem pick up that its Euler A and just doesnt fill that part of the generation data on.

**Comment by alexopus** [comment]
I tried with "Euler a" and it works. However, I had to switch to "normal" scheduler, if I do "karras" it doesn't recognize it, because Euler a karras is not a valid combo on civitai's side: https://github.com/civitai/civitai/blob/main/src/server/common/constants.ts#L124-L146

Civitai combines sampler and scheduler into a single setting, like a1111 does. And supports only a subset of possible combinations.

This node will check for a valid combo: https://github.com/alexopus/ComfyUI-Image-Saver/blob/main/nodes.py#L176-L211
I suspect that you used Euler a with Karras, in which case it would be a combo which civitai doesn't support.

I could just ignore the scheduler at this point and only fill the sampler, but I think that would be more confusing.
Rather, civitai/a1111 should split this setting, or at least support all possible combinations.
